tests/k8s/test_fields.py

```diff
@@ -39,6 +42,8 @@ class Meta:
     dict_field = Field(dict)
     _exec = Field(int)
     time_field = Field(datetime)
+    json_field = JSONField()
```

Comment:
I think it could be useful to add a test that verifies the behavior when setting JSONField to a invalid value